contents/kotlin/kotlin.md

```diff
@@ -0,0 +1,87 @@
+# Introduction to Kotlin
+
+Author: [Shradheya Thakre](https://github.com/tshradheya)
+
+**Table of Contents**
+
+* [Overview](#overview)
+* [Why Kotlin?](#why-kotlin)
+    * [Concise](#concise)
+    * [Safe](#safe)
+    * [Interoperable](#interoperable)
+    * [Tool Friendly](#tool-friendly)
+* [Kotlin for Android Apps](#kotlin-for-android-apps)
+    * [Advantages of Shifting to Kotlin](#advantage-kotlin)
+    * [Drawbacks of Shifting to Kotlin](#drawbacks-kotlin)
+
```

Comment:
Table of Contents is generally not used in other book chapters, so it might not be good to introduce a new style in this chapter on Kotlin.

I think we can have less blank spaces here?

Reply:
If it improves navigability, I don't think it's an issue. 
I don't believe there's a standard format being enforced (I could be wrong about this semester!).

Reply:
If the page is long, then having a table of contents is good. But for this case, the content is short and can be easily viewed from start to end with a single scroll (if you remove the space taken up by the table of contents and its spaces). Plus it is good to also observe the rough outline used for other chapters as mentioned.

Reply:
I think it's better to leave the ToC cause most likely there will be more content added to Kotlin and then it will be necessary